config/initializers/figaro.rb

```diff
@@ -21,7 +21,8 @@
   'GOV_DELIVERY_SERVER',
   'ES_URL',
   'ES_CLIENT_CERT_PATH',
-  'ES_CLIENT_KEY_PATH'
+  'ES_CLIENT_KEY_PATH',
+  'GIDS_HOST'
```

Comment:
Very minor nit but I'd like to normalize these config params somewhat, and I think there's slightly more parameter using `foo_URL` vs. `foo_HOST`. Can we make this `GIDS_URL` to line up.